src/viam/ur/module/ur_arm.cpp

```diff
@@ -414,8 +410,9 @@ void URArm::move_to_joint_positions(const std::vector<double>& positions, const
     const Eigen::VectorXd next_waypoint_deg = Eigen::VectorXd::Map(positions.data(), boost::numeric_cast<Eigen::Index>(positions.size()));
     const Eigen::VectorXd next_waypoint_rad = next_waypoint_deg * (M_PI / 180.0);  // convert from radians to degrees
     waypoints.push_back(next_waypoint_rad);
-    const std::chrono::milliseconds unix_time_ms = unix_now_ms();
-    auto filename = waypoints_filename(get_output_csv_dir_path(), unix_time_ms.count());
+    const auto unix_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch());
```

Comment:
I don't think you need `ms` on the variable, since it is clearly milliseconds per the duration cast, here and elsewhere.